app/packages/components/src/components/JSONPanel/panel.module.css

```diff
@@ -17,6 +17,57 @@
   pointer-events: all;
 }
 
+.lookerPanelV2 {
+  line-height: 1;
+  font-weight: bold;
+  color: var(--joy-palette-text-secondary);
+  background-color: var(--joy-palette-background-level2);
+  border: 1px solid var(--joy-palette-primary-plainBorder);
+  box-shadow: 0 8px 15px 0 var(--joy-palette-neutral-softBg);
+  border-radius: 3px;
+  overflow: auto;
+  scrollbar-width: none;
+  position: relative;
+  pointer-events: all;
+}
+
+.searchContainer {
+  position: relative;
+  display: flex;
+  padding: 1rem;
+  align-items: center;
+  background-color: var(--joy-palette-background-level2);
+  width: 90%;
+}
+
+.searchInput {
+  padding: 0.25rem 0.5rem;
+  width: 100%;
+  border-radius: 4px;
+  color: var(--joy-palette-text-secondary);
+  border: 1px solid var(--joy-palette-primary-plainBorder);
+  background-color: var(--joy-palette-background-level1);
+}
+
+.searchInput:focus {
+  border: 1px solid var(--joy-palette-primary-softBorder);
+  outline: none;
+}
+
+.searchCloseIcon {
+  position: absolute;
+  right: 1.5rem;
+  cursor: pointer;
+}
+
+.copyBtnClass {
+  font-size: 11px !important;
```

Comment:
Is `important` necessary? I really try to avoid its use

Reply:
yes! using inline style solves it